### DISCUSSION TITLE
Matching "crontab" strings from a C# attribute

### DISCUSSION THREAD

**Discussion by flcdrg** [VsixCommunity/Community.VisualStudio.Toolkit#260] [Q&A] [answered]
I had an idea to build a VSIX that interpreted the "crontab" string (eg. as used in an [Azure Function Timer Trigger](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=csharp)) and provided an explanation of the time string.

eg. the `"0 */5 * * * *"` bit of 

```csharp
[FunctionName("TimerTriggerCSharp")]
public static void Run([TimerTrigger("0 */5 * * * *")]TimerInfo myTimer, ILogger log)
{
    if (myTimer.IsPastDue)
    {
        log.LogInformation("Timer is running late!");
    }
    log.LogInformation($"C# Timer trigger function executed at: {DateTime.Now}");
}
```

I'm guessing I need to create a classifier to match the string, but just wondering what's going to be the most efficient way of doing that? Looking for the "TimerTrigger" string prefix, or can I make use of Roslyn somehow to match the attribute?

**Comment by madskristensen** [comment, marked as answer]
I think a simple regex would do the trick just fine
